index.js

```diff
@@ -1,3 +1,4 @@
+import { createCharacterCard } from "./components/card/card.js";
```

Comment:
This is going to be a default export (see my comment above), so you will need to adjust this import too:
```suggestion
import createCharacterCard from "./components/card/card.js";
```